docs/configure-netlify-redirects.adoc

```diff
@@ -35,7 +35,8 @@ More details about redirects handling in the https://cli.netlify.com/netlify-dev
 
 === Step-by-step procedure
 
-- Build a preview with the components and versions needed for the test. See xref:../README.adoc#preview-support[README]
+- Build a preview with the components and versions needed for the test. See xref:../README.adoc#preview-support[README]. +
+Add the `--type netlify` parameter to use the configuration for the whole Netlify environment.
```

Comment:
**suggestion:** let explain what to do instead on how to do it. This will simplify the maintenance
```suggestion
Don't forget to build the preview to target the `Netlify` environment.
```

Reply:
Ignore, this is good like this. The maintenance cost is very low (we probably won't change the option).